geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/iterators/Z2DensityIterator.scala

```diff
@@ -65,8 +65,8 @@ class Z2DensityIterator extends KryoLazyDensityIterator {
   /**
    * We write the geometry at the center of the zbox that this row represents
    */
-  override def writeNonPoint(geom: Geometry, weight: Double, result: DensityResult): Unit = geom match {
-    case p: Point => writePointToResult(p, weight, result)
+  def writeNonPoint(geom: Geometry, weight: Double, result: DensityResult): Unit = geom match {
```

Comment:
hmm, are these methods actually getting invoked anymore? seems like it was overriding methods in KryoLazyDensityUtils and now it's not